CHANGELOG.md

```diff
@@ -31,6 +31,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - New internal module to hanle `.tar` files.
 
+- New GraphQL query `{ webui_blacklist }`.
```

Comment:
This is meaningless from the point of view of a user. User now has an ability to hide some webui page.